dipy/reconst/recspeed.pyx

```diff
@@ -57,6 +58,8 @@ def remove_similar_vertices(
     return_indices : {False, True}, optional
         If True, return `indices` as well as `vertices` and maybe `mapping`
         (see below).
+    is_symmetric : {False, True}, optional
```

Comment:
Since this is in independent function, I suggest renaming the parameter `is_symmetric` to `remove_antipodale`